doc/preprod_update.rst

```diff
@@ -0,0 +1,74 @@
+Updating the preprod Server
+===========================
+
+This method is a temporary method in order to quickly update the preprod server with the latest version of the code.
+
+Preprod server has an assembl instance running on different containers. The whole instance is run via a .yml file run with docker-compose
+
+
+Note: 6f41608ee826 is the identity of the container running the image assembl:develop
```

Comment:
I wouldn't do this. The next time the image is rebuilt, the docs will have to change to represent this new container ID. I would only describe that the container id can be found using `docker ps | grep assembl`